src/components/Home/index.css

```
@@ -1,69 +1,28 @@
+.flex_container {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-evenly;
+  align-items: center;
+}
+
+.flex_item_description {
+  align-items: center;
+  padding-top: 150px;
+}
+
 .background {
   width: 100%;
   height: calc(100vh - 56px);
-  position: fixed;
   background-image: url(../../images/home.jpg);
   background-size: cover;
 }
 
 .buttonstyle {
-  padding: 20px 100px 380px 100px;
+  width: 20%;
+  height: 50%;
   border-radius: 20px;
   font-style: Helvetica;
```

This is invalid for font-style.
```suggestion
  font-family: Helvetica, Sans-Serif;
```